### PR TITLE
Forward-porting fileinfo limits

### PR DESCRIPTION
**Pull request by harshilsharma63**
Ported file info limits code to main.

### CODE REVIEW

File: server/services/store/sqlstore/migrations/000017_add_teams_and_boards.up.sql [hunk 1]
```diff
@@ -13,6 +13,7 @@ ALTER TABLE {{.prefix}}blocks_history ADD COLUMN board_id VARCHAR(36);
 {{- /* cleanup incorrect data format in column calculations */ -}}
 {{- /* then move from 'board' type to 'view' type*/ -}}
 {{if .mysql}}
+SELECT 'I am mysql';
```
Comment (jespino) [inline server/services/store/sqlstore/migrations/000017_add_teams_and_boards.up.sql:16]: debug strings here

File: webapp/src/octoClient.ts [hunk 2]
```diff
@@ -556,18 +556,23 @@ class OctoClient {
...
+        if (response.status === 200) {
+            const blob = await response.blob()
+            fileURL.url = URL.createObjectURL(blob)
+        } else if (response.status === 400) {
+            fileURL = await this.getJson(response, {}) as FileInfo
         }
-        const blob = await response.blob()
-        return URL.createObjectURL(blob)
+
+        return fileURL
```
Comment (jespino) [inline webapp/src/octoClient.ts:575]: nit: a bit misleading name of variable.

**Review by jespino — APPROVED** [review]
LGTM

File: server/services/store/sqlstore/migrations/000017_add_teams_and_boards.up.sql (hunk 1 above)
Comment (harshilsharma63) [reply server/services/store/sqlstore/migrations/000017_add_teams_and_boards.up.sql]: Done.

**Pull request merged**